### ISSUE TITLE
Can't drop file with `--target`

### ISSUE THREAD

**Issue by IFcoltransG** [nik012003/ripdrag#14]
On Manjaro Linux with i3 and `ripdrag v0.2.0`, running `ripdrag --target` and dragging a file from a file manager (`pcmanfm-qt`) does nothing. After that, dragging more files (or strings) prints the following to the console:
```
(ripdrag:404510): Gtk-WARNING **: 12:31:07.362: (../gtk/gtk/gtkdroptarget.c:477):gtk_drop_target_handle_crossing: runtime check failed: (self->drop == NULL || self->drop == crossing->drop)

(ripdrag:404510): Gtk-CRITICAL **: 12:31:07.362: gtk_drop_target_handle_event: assertion 'self->drop == gdk_dnd_event_get_drop (event)' failed
```
Both of these messages are repeated many times. The former whenever I move the mouse over the popup, and the latter whenever I drop something onto it, possibly other times.

Before dropping the first file, nothing is printed and I am able to drop strings correctly. After dropping the first file, all dragging and dropping of files or strings fails in the above way.

`--verbose` does not seem to change the output.

**Comment by nik012003** [comment]
Thank you for reporting! 
Unfortunately I still haven't found a way around this problem. 
At this moment in time I'm only able to drag and drop strings, in particular those with "text/uris-list" mime type. So if your file manager supports that, it should work with ripdrag.
I will try to fix this for the next release, but I think that requires some modifications in gtk4-rs and it will take time.

**Comment by nik012003** [comment]
Solved in version v0.3.0
